frontend/src/metabase/components/AppBanner/AppBanner.tsx

```diff
@@ -0,0 +1,28 @@
+import type { Location } from "history";
+
+import { useSelector } from "metabase/lib/redux";
+import { getSetting } from "metabase/selectors/settings";
+import { getUserIsAdmin } from "metabase/selectors/user";
+import {
+  PaymentBanner,
+  shouldRenderPaymentBanner,
+} from "metabase/nav/components/PaymentBanner/PaymentBanner";
+import { DatabasePromptBanner } from "metabase/nav/components/DatabasePromptBanner";
+
+interface AppBannerProps {
+  location: Location;
+}
+
+export const AppBanner = ({ location }: AppBannerProps) => {
+  const isAdmin = useSelector(getUserIsAdmin);
+  const tokenStatusStatus = useSelector(
+    state => getSetting(state, "token-status")?.status,
+  );
+  if (shouldRenderPaymentBanner({ isAdmin, tokenStatusStatus })) {
```

Comment:
Only render one banner at a time, and if somehow both banner conditions are met, only render the payment banner. [Discussion here.](https://metaboat.slack.com/archives/C057T1QTB3L/p1685549180307149?thread_ts=1685547913.580079&cid=C057T1QTB3L)